src/Symfony/Component/VarDumper/Caster/FFICaster.php

```diff
@@ -29,8 +29,10 @@ final class FFICaster
      *
      * This restriction serves to ensure that processing does not take
      * up the entire allowable PHP memory limit.
+     *
+     * @internal
      */
-    private const MAX_STRING_LENGTH = 255;
+    public const MAX_STRING_LENGTH = 255;
```

Comment:
that's not worth opening for a test case, better duplicate

Reply:
Right